src/http.rs

```diff
@@ -37,6 +37,10 @@ use crate::{
     tls::{tls_connector_builder, MaybeTlsSettings, TlsError},
 };
 
+pub mod status {
+    pub const TOO_MANY_REQUESTS: u16 = 429;
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** Nice